.golangci.yml

```diff
@@ -53,15 +53,12 @@ issues:
         - stylecheck
 
     # TODO: Remove these when we are ready to drop support for Legacy Amino
-    - text: "SA1019: codec.LegacyAmino is deprecated"
+    - text: 'SA1019: "github.com/cosmos/cosmos-sdk/types/bech32/legacybech32" is deprecated'
```

Comment:
This has changed due to the fact that with the new linting process, the warning has changed as well